Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs

```diff
@@ -760,10 +772,21 @@ private async Task RunObserverAsync(CancellationToken cancellationToken)
         /// Gets the matrix representing the transform from scene space to world space
         /// </summary>
         /// <returns>The transform matrix</returns>
-        private System.Numerics.Matrix4x4 GetSceneToWorldTransform()
+        private System.Numerics.Matrix4x4? GetSceneToWorldTransform()
         {
             var result = System.Numerics.Matrix4x4.Identity;
 #if WINDOWS_UWP
+#if MSFT_OPENXR
+            SpatialGraphNode node = SpatialGraphNode.FromStaticNodeId(sceneOriginId);
+            if (node.TryLocate(FrameTime.OnUpdate, out Pose pose))
+            {
+                result = Matrix4x4.TRS(pose.position, pose.rotation, Vector3.one).ToSystemNumerics();
+            }
+            else
+            {
+                return null;
```

Comment:
Do we want to either:

1. Return `System.Numerics.Matrix4x4.Identity` here to match the `#else !MSFT_OPENXR` case
2. or update the `#else !MSFT_OPENXR` case to return null `if (!sceneToWorld.HasValue)`

Reply:
Could check the return value on line 669 against `System.Numerics.Matrix4x4.Identity` instead of against null if option 1

Reply:
Just took the second route.